packages/ses/src/commons.js

```diff
@@ -135,4 +199,34 @@ export const getConstructorOf = fn =>
  * immutableObject
  * An immutable (frozen) exotic object and is safe to share.
  */
-export const immutableObject = freeze({ __proto__: null });
+export const immutableObject = freeze(create(null));
+
+/**
+ * isObject tests whether a value is an object.
+ * Today, this is equivalent to:
+ *
+ *   const isObject = value => {
+ *     if (value === null) return false;
+ *     const type = typeof value;
+ *     return type === 'object' || type === 'function';
+ *   };
+ *
+ * But this is not safe in the face of possible evolution of the language, for
+ * example new types or semantics of records and tuples.
+ * We use this implementation despite the unnecessary allocation implied by
+ * attempting to box a primitive.
+ *
+ * @param {any} value
+ */
+export const isObject = value => Object(value) === value;
```

Comment:
More pork. Not all uses of `isObject` were previously defined this way, but as we discussed out-of-band, they all should work with this generalization, which favors integrity in the face of language evolution over memory pressure performance.